dist/formats/finder_email_signup/frontend/schema.json

```diff
@@ -320,6 +320,19 @@
               },
               "filter_value": {
                 "type": "string"
+              },
+              "option_lookup": {
+                "description": "A map of keys to values to associate a displayed value with multiple options",
```

Comment:
I'm not sure this is super clear, without more context it's not clear what we're mapping keys to/from, where they're displayed or why we might want to do this.

Reply:
I've changed the text and regenerated the schemas. Let me know if the new text is OK :)

Reply:
Perhaps it's just the tiredness but based on this description I'm still not sure what this does (and I _think_ it's probably a good idea to make it clear here as I'm not sure where else people will be able to see what it does). Let's chat about it tomorrow mornign

Reply:
Sure thing. The current text matches that of the other `option_lookup` for finder facets: https://github.com/alphagov/govuk-content-schemas/pull/867/commits/d06f82b4b3abb43a898f52b587464adf13c7f0c6#diff-6cf8ee2493a65fcb847de8a4dbf8e0daR195.

Reply:
I'm happy to chat about this too, as I was the one who wrote the original description and perhaps thought that it could be clearer.

Reply:
Thanks @karlbaker02, see https://github.com/alphagov/govuk-content-schemas/pull/868#issuecomment-472045661